dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -136,13 +136,21 @@ export const ManyNewsletterSignUp = () => {
 			if (!(button instanceof HTMLElement)) {
 				return;
 			}
-			const id = button.getAttribute('data-newsletter-id');
-			if (!id) {
+			const identityName = button.getAttribute('data-newsletter-id');
```

Comment:
That is neater, but I think i'd prefer to to use `getAttribute`. I can imagine my future self seeing the code that sets the attribute in the other component, searching the codebase for other instances of  "data-identity-name" (attribute name changed as per other comment) and concluding we don't need the attribute because nowhere consumes it, having forgotten that `Element.dataset` is a possibility.